package/entrypoint.sh

```diff
@@ -12,10 +11,12 @@ if [ -e /var/lib/rancher/etcd ] && [ ! -e /var/lib/rancher/k3s/server/db/etcd ];
   echo -n 'default' > /var/lib/rancher/k3s/server/db/etcd/name
 fi
 if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  echo "INFO: Running k3s server --cluster-init --cluster-reset"
   k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
   if [ $? -ne 0 ]; then
     echo "ERROR:" && cat ./k3s-cluster-reset.log
     rm -f /var/lib/rancher/k3s/server/db/reset-flag
+    exit $?
```

Comment:
Unless your intent is to return the exit code of `rm -f` I think we need something like:
```suggestion
  k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
  K3S_EXIT=$?
  if [ $K3S_EXIT -ne 0 ]; then
    echo "ERROR:" && cat ./k3s-cluster-reset.log
    rm -f /var/lib/rancher/k3s/server/db/reset-flag
    exit $K3S_EXIT
```

Reply:
Yep agree with this. Thanks for taking a look @brandond

Reply:
true, I rushed this PR up without testing it. I’m not sure this is the best approach in the first place though (getting rid of the set -e) so I need to think about it some more